scripts/release_ppa.sh

```diff
@@ -47,7 +47,7 @@ set -e
 
 REPO_ROOT="$(dirname "$0")/.."
 
-# for the "fail" function
+# Import "fail" and "checkDputEntries"
```

Comment:
```suggestion
# Import "fail", "checkDputEntries" and "sourcePPAConfig"
```
But I'm not sure if it's a good idea to have these listed here. The list will easily go out of sync.